scripts/install.js

```diff
@@ -48,7 +48,7 @@ function download(url, dest, cb) {
     return response.statusCode >= 200 && response.statusCode < 300;
   };
 
-  console.log('Start downloading binary at', url);
+  log.http('node-sass install', 'Start downloading binary at', url);
```

Comment:
The `, url` doesn't work the same way as it did with `console`, so it either needs a temaplate in the message string or just switch it to `+ url`